inc/migration.class.php

```diff
@@ -667,6 +670,14 @@ function migrationOneTable($table) {
          unset($this->fulltexts[$table]);
       }
 
+      if (isset($this->uniques[$table])) {
+         $this->displayMessage( sprintf(__('Adding unicity index - %s'), $table));
```

Comment:
```suggestion
         $this->displayMessage( sprintf(__('Adding unicity indexes - %s'), $table));
```

Reply:
As default language is en_GB, maybe should we use `indices` («This usage is gaining acceptance in common usage in the U.S., as opposed to the Latin-derived form indices, often used in the UK.» - https://en.wiktionary.org/wiki/indexes#English), see also https://grammarist.com/usage/indexes-indices/